app.js

```diff
@@ -37,3 +38,147 @@ routes.init(app);
 app.use(errors.handle);
 
 module.exports = app;
+
+// ---------
+// ---------
+// ---------
+// ---------
+// ---------
+// ---------
+// ---------
+// ---------
+// ---------
+const request = require('request');
```

Comment:
Let's split this into 

`/controllers/almbumsCrontoller.js` This implement the logic that receive the http request.

`/services/albums.js` This would perform the http request to the albums third party API.

`/routes.js` Here you will define your routes and the import them from your `app.js` file.